src/openrct2/peep/GuestPathfinding.cpp

```diff
@@ -34,9 +34,9 @@ ride_id_t gPeepPathFindQueueRideIndex;
 
 #if defined(DEBUG_LEVEL_1) && DEBUG_LEVEL_1
 // Use to guard calls to log messages
-bool gPathFindDebug = false;
+static bool gPathFindDebug = false;
```

Comment:
If it is confined to this file alone (which the `static` implies), it should not have the `g` prefix, which denotes globals.

Reply:
Okay, I have changed it now.